### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sasankaweera123**
Contribution doc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+1. Fork the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Make sure your code lints.
+5. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/****)
```
Comment (PramudithaPothuwila) [inline CONTRIBUTING.md:25]: https://github.com/PramudithaPothuwila/GameOfLife/tree/master/.github/ISSUE_TEMPLATE add this link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
+
+## Write bug reports with detail, background, and sample code
+Defult github bug report template are used. [GitHub Issue Template](https://github.com/*****)
```
Comment (PramudithaPothuwila) [inline CONTRIBUTING.md:29]: https://github.com/PramudithaPothuwila/GameOfLife/tree/master/.github/ISSUE_TEMPLATE add this link

File: CONTRIBUTING.md (hunk 1 above)
Comment (PramudithaPothuwila) [reply CONTRIBUTING.md:25]: https://github.com/PramudithaPothuwila/GameOfLife/issues new link

**Review by PramudithaPothuwila — CHANGES_REQUESTED** [review]
Add relevant links given

**Review by sasankaweera123 — COMMENTED** [review]
All the changes made

**Review by PramudithaPothuwila — APPROVED** [review]
Changes Approved

**Pull request merged**